### DISCUSSION TITLE
CC2531 crashes after while

### DISCUSSION THREAD

**Discussion by evaanp** [Koenkk/zigbee2mqtt#8377] [General]
Hi,

I'm currently using the CC2531 usb dongle to communicate with mostly GU10 ikea bulbs. I have 33 devices on my network. After reflashing it works for around two days before i get the mgmtPermitJoinReq 6000 ms error. The only way to make it work again is to reflash the dongle.

I'm considering to buy the zzh! CC2652R Multiprotocol RF Stick, but it is currently sold out and it is a bit more expensive than the CC2531. Is the problem indeed solved by buying the CC2652R stick? Or are there other things i could try. Is there any change that the problem is solved by flashing the 3.x.x firmware instead of the 1.2 firmware?

Kind regards,
Evan

**Comment by lux73** [comment]
there's another CC2652 based Stick available: https://slae.sh/projects/cc2652/

after switching from CC2531 to the CC2652RB all my Problems are gone! i can recommend switching!

my old CC2531 had Problems with even less Devices -> after 19 Devices my Stick had similar drop-outs...

**Reply by evaanp** [reply]
Thanks!

**Comment by evaanp** [comment]
Update: received the stick today and it's working great so far. Didn't even had to re-pair my devices.
